PennMobile/New Group/AccountPageViewController.swift

```diff
@@ -0,0 +1,132 @@
+//
+//  AccountPageViewController.swift
+//  PennMobile
+//
+//  Created by Andrew Antenberg on 9/26/21.
+//  Copyright © 2021 PennLabs. All rights reserved.
+//
+
+import Foundation
+import UIKit
+
+class AccountPageViewController: UIViewController, ShowsAlertForError, UITableViewDelegate, UITableViewDataSource {
+    var account: Account!
+    let tableView = UITableView(frame: .zero, style: .insetGrouped)
+    
+    var profileInfo = [(text: "Name", info: " "), (text: "Username", info: " "), (text: "Email", info: " ")]
```

Comment:
Rather than initializing them with missing values, it would be ideal for you to build them up in the method below by appending them. 

`var profileInfo: [(text: String, info: String)] = []`